src/Akka.HealthCheck.Tests/Transports/SocketStatusTransportSpecs.cs

```diff
@@ -73,16 +80,16 @@ public async Task Should_successfully_repeatedly_open_signal()
             var result2 = await Transport.Go("bar", CancellationToken.None);
             result.Success.Should().BeTrue();
 
-            var bytesRead = tcpClient.Available;
-            bytesRead.Should().Be(0);
+            
+            AwaitAssert(()=> tcpClient.Available.Should().Be(8));
             tcpClient.Connected.Should().BeTrue();
 
             // special case - need to test the NULL pattern
             var result3 = await Transport.Go(null, CancellationToken.None);
             result.Success.Should().BeTrue();
 
-            bytesRead = tcpClient.Available;
-            bytesRead.Should().Be(0);
+            
+            AwaitAssert(() => tcpClient.Available.Should().Be(8)); 
```

Comment:
@izavala shouldn't this be 0 bytes read?

Reply:
It should read bytes as it should be getting a response, the tcpClient.Available is still active